prometheus/client.go

```diff
@@ -43,6 +44,14 @@ type Workload struct {
 	Version   string
 }
 
+// Service describes a service with contextual information
+type Service struct {
+	Namespace   string
+	App         string
+	ServiceName string
+	Service     string
```

Comment:
What "Service" stands for? is it like the full qualified name "svc.namespace.cluster"? This is weird to see both "Service" and "ServiceName" here